internal/evaluator/conftest_evaluator.go

```diff
@@ -71,6 +71,7 @@ const (
 	metadataEffectiveOn = "effective_on"
 	metadataTerm        = "term"
 	metadataTitle       = "title"
+	EffectiveOnStale    = 90
```

Comment:
I think it should be lowercase unless you want it visible in other modules.
```suggestion
	effectiveOnStale    = 90
```

Reply:
Good point, fixed.

Reply:
Maybe a comment:

```suggestion
	effectiveOnStale    = 90 # Days
```

Or make it more clear like this:

```suggestion
	effectiveOnStaleDays  = 90
```

Reply:
I like the second option better, but it does push the spacing for the entire `const` a little bit. Still okay?

Reply:
> I like the second option better, but it does push the spacing for the entire `const` a little bit. Still okay?

That shouldn't be a concern.